test/is_utility_stmt.c

```diff
@@ -0,0 +1,127 @@
+#include <pg_query.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "test/framework/main.h"
+
+extern PgQueryIsUtilityResult pg_query_is_utility_stmt(const char *query);
```

Comment:
Once we add this to `pg_query.h` (see separate comment) it should not be necessary to declare it here.